src/Simplex/Chat/Core.hs

```diff
@@ -44,3 +70,59 @@ sendChatCmdStr cc s = runReaderT (execChatCommand Nothing . encodeUtf8 $ T.pack
 
 sendChatCmd :: ChatController -> ChatCommand -> IO ChatResponse
 sendChatCmd cc cmd = runReaderT (execChatCommand' cmd) cc
+
+getSelectActiveUser :: SQLiteStore -> IO (Maybe User)
+getSelectActiveUser st = do
+  users <- withTransaction st getUsers
+  case find activeUser users of
+    Just u -> pure $ Just u
+    Nothing -> selectUser users
+  where
+    selectUser :: [User] -> IO (Maybe User)
+    selectUser = \case
+      [] -> pure Nothing
+      [user@User {userId}] -> do
+        withTransaction st (`setActiveUser` userId)
+        pure $ Just user
+      users -> do
+        putStrLn "Select user profile:"
+        forM_ (zip [1 :: Int ..] users) $ \(n, user) -> putStrLn $ show n <> ": " <> userStr user
+        loop
+        where
+          loop = do
+            nStr <- getWithPrompt $ "user number (1 .. " <> show (length users) <> ")"
+            case readMaybe nStr :: Maybe Int of
+              Nothing -> putStrLn "not a number" >> loop
+              Just n
+                | n <= 0 || n > length users -> putStrLn "invalid user number" >> loop
+                | otherwise -> do
+                    let user@User {userId} = users !! (n - 1)
+                    withTransaction st (`setActiveUser` userId)
+                    pure $ Just user
+
+createActiveUser :: ChatController -> IO User
+createActiveUser cc = do
+  putStrLn
+    "No user profiles found, it will be created now.\n\
+    \Please choose your display name and your full name.\n\
+    \They will be sent to your contacts when you connect.\n\
+    \They are only stored on your device and you can change them later."
```

Comment:
```suggestion
    "No user profiles found, it will be created now.\n\
    \Please choose your display name.\n\
    \It will be sent to your contacts when you connect.\n\
    \It is only stored on your device and you can change it later."
```